EIPS/eip-draft_alias_origin_to_sender.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7645
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.